src/clj/puppetlabs/jdbc_util/core.clj

```diff
@@ -105,3 +107,17 @@
                        [sql 0]
                        seq-params-w-indices)]
     (vec (conj (flatten parameters) sql'))))
+
+;; Connection pooling
+
+(s/defn ^:always-validate pooled-db-spec :- cp/pooled-db-spec-schema
```

Comment:
I put these both here, since it felt better to proxy the functions from core but I'm not beholden to this approach.